en/docs/try-out/payment-initiation-service-flow.md

```diff
@@ -0,0 +1,601 @@
+This page provides instructions to use the NextGenPsd2XS2AFramework API to provide the Payment Initiation Service (PIS).
+
+!!! tip "Before you begin:"
+    Deploy the [NextGenPsd2XS2AFramework API](deploy-nextgenpsd2-api.md). 
```

Comment:
capitalize PSD

Reply:
Fixed in 7bfa39b2fba54a9ecb39b651ec24854dea85354f